src/dft/src/clock_domain/ClockDomain.cpp

```diff
@@ -68,4 +68,26 @@ ClockEdge ClockDomain::getClockEdge() const
   return clock_edge_;
 }
 
+std::string_view ClockDomain::getClockEdgeName() const
+{
+  switch (clock_edge_) {
+    case ClockEdge::Rising:
+      return "rising";
+      break;
+    case ClockEdge::Falling:
+      return "falling";
+      break;
+    default:
+      // TODO replace with std::unreachable() once we reach c++23
+      return "";
+      break;
```

Comment:
Its best not to use a default case with an enum that way the compiler will complain if any case is unused

Reply:
Right. I changed this to use the tip from here: [abseil.io/tips/147](https://abseil.io/tips/147)

We need to return something anyway, otherwise the compiler with complain with "control reaches end of non-void function "